conf/test_alternativepreprocessing.config

```diff
@@ -10,15 +10,19 @@
 ----------------------------------------------------------------------------------------
 */
 
+process {
+    // Limit resources so that this can run on GitHub Actions
+    resourceLimits = [
+        cpus: 2,
+        memory: '6.GB',
```

Comment:
Should be synced with teh `test.config` values, e.g. memory should be `15.GB` etc

And applies across all configs

Reply:
I will update the memory limit in all test configs to 15. GB.